docs/source/configuration/environmentvariables.md

```diff
@@ -93,21 +93,71 @@ It will enable the log of missing i18n messages (in console).
 ### Add add-ons via environment variable
 
 You can use the `ADDONS` environment variable to enable and configure add-ons in your project.
-They sum up to the ones set fixed in `package.json` or programatically in `volto.config.js`.
-You can specify released (published) packages (installed previously in your environment, and present already in `node_modules`), or `packages` folder local to your project (not in development, but optional, for example, Volto's testing packages).
 It can be used to temporarily add an add-on to your build for testing purposes.
 It can also be used to temporarily enable a feature or a set of customizations.
 
+When running your app, the set of add-ons that will be loaded is the union of the add-ons defined in this three locations, in this order:
```

Comment:
"union of the addons" seems restrictive. Maybe we can implement an export in the volto.config.js, something like modifyAddons? Or maybe the ADDONS env var could work as an "all or nothing", something that would not be added, but would override everything else.

Reply:
@tiberiuichim yeah, I simplified the wording after looking at @sneridagh's feedback. Please see the latest language in my latest commit.

https://github.com/plone/volto/pull/3067/commits/57488a060599237dbbc4e0a8601eb044e555d220

Reply:
@tiberiuichim I think that the ADDONS like it's now accomplish the "all or nothing" as well, since optionally you can remove then all your package.json and volto.config.js addons, and only rely on the ADDONS env var. However, a mean to accomplish what you proposed could be also useful (maybe a flag? to state only to add the env var ones?)